components/atoms/TopContributorCard/top-contributor-card.tsx

```diff
@@ -11,9 +11,10 @@ import Button from "../Button/button";
 
 export interface TopContributorCardProps {
   login: string;
+  mutateLoggedInUser: () => void;
```

Comment:
nit: This naming is pretty specific to SWR

```suggestion
  refreshLoggedInUser: () => void;
```

Reply:
@brandonroberts Okay, I'll change it